CorsixTH/Lua/objects/reception_desk.lua

```diff
@@ -207,7 +226,7 @@ function ReceptionDesk:onDestroy()
       end
     end)
   end
-  self.queue:rerouteAllPatients(SeekReceptionAction())
+  self.queue:rerouteAllPatients(SeekReceptionAction(), object)
```

Comment:
There is no object still?

Reply:
Object is declared at line 21. It passes that into the function as the "origin". I tried using self but it just returns nil.